showcase/app/templates/components/application-state.hbs

```diff
@@ -7,124 +7,494 @@
+        <A.Header @title="Empty state title" />
+        <A.Body
+          @text="Sorry, an unexpected error has occurred. Please try again later or contact support for assistance."
+        />
+        <A.Footer @hasDivider={{true}} as |F|>
+          <F.Dropdown @listPosition="bottom-right" as |dd|>
+            <dd.ToggleButton @text="Choose an option" />
+            <dd.Title @text="Categories" />
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Changelogs" @href="#" />
+          </F.Dropdown>
         </A.Footer>
       </Hds::ApplicationState>
     </SF.Item>
-    <SF.Item @label="Empty state, icon in header">
+    <SF.Item @label="Dropdown + button as actions">
       <Hds::ApplicationState as |A|>
-        <A.Header @title="Empty state title text" @icon="help" />
-        <A.Body>
-          <Shw::Placeholder @text="components and other things" @height="150px" @background="#e1f5fe" />
-        </A.Body>
-        <A.Footer as |F|>
-          <F.LinkStandalone @icon="arrow-left" @text="Go back" @href="/" />
+        <A.Header @title="Empty state title" />
+        <A.Body
+          @text="Sorry, an unexpected error has occurred. Please try again later or contact support for assistance."
+        />
+        <A.Footer @hasDivider={{true}} as |F|>
+          <F.Dropdown @listPosition="bottom-right" as |dd|>
+            <dd.ToggleButton @text="Choose an option" />
+            <dd.Title @text="Categories" />
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Changelogs" @href="#" />
+          </F.Dropdown>
+          <F.Button @color="secondary" @text="Secondary action" />
+        </A.Footer>
+      </Hds::ApplicationState>
+    </SF.Item>
+    <SF.Item @label="Dropdown + button + link as actions">
+      <Hds::ApplicationState as |A|>
+        <A.Header @title="Empty state title" />
+        <A.Body
+          @text="Sorry, an unexpected error has occurred. Please try again later or contact support for assistance."
+        />
+        <A.Footer @hasDivider={{true}} as |F|>
+          <F.Dropdown @listPosition="bottom-right" as |dd|>
+            <dd.ToggleButton @text="Choose an option" />
+            <dd.Title @text="Categories" />
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Changelogs" @href="#" />
+          </F.Dropdown>
+          <F.Button @color="secondary" @text="Secondary action" />
+          <F.LinkStandalone @icon="docs-link" @text="Learn more" @iconPosition="trailing" @href="#" />
         </A.Footer>
       </Hds::ApplicationState>
     </SF.Item>
   </Shw::Flex>
+
+  <Shw::Divider />
+
+  <Shw::Text::H2>Alignment</Shw::Text::H2>
```

Comment:
I would add also at least one use case (left + center) where one of the actions is a dropdown